client/jetpack-cloud/sections/partner-portal/primary/payment-method-add/index.tsx

```diff
@@ -73,7 +78,28 @@ function PaymentMethodAdd() {
 		[ window.location.href, getQueryArg ]
 	);
 
+	const products = useMemo(
+		() => ( getQueryArg( window.location.href, 'products' ) || '' ).toString(),
+		[]
+	);
+
+	const siteId = useMemo(
+		() => getQueryArg( window.location.href, 'site_id' ) || '',
+		[]
+	).toString();
+
+	const source = useMemo(
+		() => getQueryArg( window.location.href, 'source' ) || '',
+		[]
+	).toString();
+
+	const isMulti = isEnabled( 'jetpack/partner-portal-issue-multiple-licenses' );
```

Comment:
Can we give a more meaningful name to this variable? For example, `isMultipleLicenseIssueFlow`?